src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -1207,6 +1229,7 @@ impl CatalogState {
             .cluster_replica_sizes
             .0
             .iter()
+            .filter(|(_, a)| !a.disabled)
```

Comment:
This is present in at least 3 places (`.cluster_replica_sizes.0.iter().filter(|(_, a)| !a.disabled)`). Can it get turned into a named method on something to assist us in correctly using it? Otherwise it's super easy to accidentally not filter when writing new code.

Reply:
Introduced a method to encapsulate this logic!